packaging/centos-systemd/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:8
```

Comment:
Are these dockerfiles just for the `test-packages` target? It'd be useful to future maintainers if there was a comment here or if the file was named `Dockerfile.test` (or both!)

(Ditto for the other new dockerfiles)

Reply:
I think comments would be really useful. The `fpm` image is a kind of `build-image` for packaging, the `{debian,centos}-systemd` images are just for testing the packages out since my Linux distribution has a different package manager.